ingest/validation/validate_metadata.py

```diff
@@ -307,12 +322,29 @@ def cast_float_type(value):
 def cast_string_type(value):
     """Cast string type per convention where Pandas autodetected a number
     """
-    if isinstance(value, numbers.Number):
+    if value_is_nan(value):
+        # nan indicates missing data; by type, nan is a numpy float
+        # so a separate type check is needed for proper handling
+        return value
+    elif isinstance(value, numbers.Number):
         return str(value)
     else:
         return value
 
 
+def regularize_ontology_id(value):
+    """Regularize ontology_ids for storage with underscore format
+    """
+    try:
+        ontology_shortname, term_id = re.split('[_:]', value)
```

Comment:
I think it would be easier instead of splitting and rejoining to just do
`return value.replace(":", "_")`

Reply:
great suggested, implemented in 3ad4b5d